depends/README.md

```diff
@@ -41,7 +41,13 @@ The paths are automatically configured and no other options are needed.
 
 #### Common
 
-    apt install bison cmake curl make patch pkg-config python3 xz-utils
+    apt install cmake curl make patch
+
+#### GUI
+
+Skip the following packages if you don't intend to use the GUI and will build with [`NO_QT=1`](#dependency-options):
+
+    apt install bison pkg-config python3 xz-utils
```

Comment:
I suggest either leaving out `apt install` or also naming `brew install` and friends, to make it clear this applies to all platforms.

Reply:
I wouldn't expect that all packages have the same names across Ubuntu/Debian and Homebrew repositories.

I agree that a macOS-specific section can be added here, but this is not a goal of this PR.

Reply:
I've double-checked prerequisites required for building depends on macOS, and there are no additional requirements specific to depends beyond those listed in https://github.com/bitcoin/bitcoin/blob/master/doc/build-osx.md#3-install-required-dependencies.